ethereum/core/src/test/java/org/hyperledger/besu/ethereum/difficulty/fixed/FixedProtocolScheduleTest.java

```diff
@@ -56,4 +56,36 @@ public void reportedDifficultyForAllBlocksIsAFixedValue() {
                 .nextDifficulty(1, parentHeader, null))
         .isEqualTo(FixedDifficultyCalculators.DEFAULT_DIFFICULTY);
   }
+
+  @Test
+  public void reportedDifficultyForAllBlocksIsAFixedValueKeccak() {
+
+    final ProtocolSchedule schedule =
+        FixedDifficultyProtocolSchedule.create(GenesisConfigFile.ecip1049dev().getConfigOptions());
+
+    final BlockHeaderTestFixture headerBuilder = new BlockHeaderTestFixture();
+
+    final BlockHeader parentHeader = headerBuilder.number(1).buildHeader();
+
+    assertThat(
+            schedule
+                .getByBlockNumber(0)
+                .getDifficultyCalculator()
+                .nextDifficulty(1, parentHeader, null))
+        .isEqualTo(FixedDifficultyCalculators.DEFAULT_DIFFICULTY);
+
+    assertThat(
+            schedule
+                .getByBlockNumber(500)
+                .getDifficultyCalculator()
+                .nextDifficulty(1, parentHeader, null))
+        .isEqualTo(FixedDifficultyCalculators.DEFAULT_DIFFICULTY);
+
+    assertThat(
```

Comment:
Great, does one of these tests cover the case that the difficulty at a specifc block the Proof of Work can be increased to an arbitrary number?

Reply:
I'm not sure. The work took place at a lower level than that. I haven't had to deal with difficulty adjustments. I'll look.

Reply:
Checked - difficulty calculations are here: https://github.com/hyperledger/besu/blob/01c02e074196ed6b4fab951725787e40d2a8f097/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ClassicDifficultyCalculators.java#L68
The difficulty calculator would be the same as ethash, and as you can see, can be tweaked as we learn more.